pkg/candlepin_client/manifest.go

```diff
@@ -0,0 +1,42 @@
+package candlepin_client
+
+import (
+	"fmt"
+	"os"
+
+	caliri "github.com/content-services/caliri/release/v4"
+)
+
+func (c *cpClientImpl) ImportManifest(filename string) error {
+	file, err := os.Open(filename)
```

Comment:
probably want to do a `defer file.Close()`

Reply:
great catch!

Reply:
I see a rebase, but I don't see a change here. Did you forget to update this?

Reply:
arg, forgot to pop the stash after the rebase, thanks!